src/Transform/Monomorphize.hs

```diff
@@ -212,7 +215,7 @@ monomorphizeFile (SProgs sps stm) =
   let dm = makeDefMap sps
       tpms = makeTypeParams sps
       xis = makeEmptyInsts sps
-      xis' = foldr (\ (x, tis) xis -> addInsts dm tpms xis x tis) xis (collectCalls stm)
+      xis' = foldr (\ (x, tis, as) xis -> addInsts dm tpms xis x tis as) xis (collectCalls stm)
```

Comment:
Shouldn't these be called `(x, tgs, tis)`?

Reply:
Yes, give me one sec...